main.py

```diff
@@ -103,17 +103,14 @@
     elif cmd == "beta":
         print (servers[str(message.server.id)]["beta_features"])
         # toggle beta command
-        with servers[str(message.server.id)]["beta_features"] as toggle_state:
-            if message.author.permissions.manage_server:
-                # has perms to toggle
-                # switch it to the state it isn't (toggle it)
-                if toggle_state:
-                    toggle_state = False
-                else:
-                    toggle_state = True
-            else:
-                # doesnt have perms
-                await client.send_message(message.channel, "**Sorry, but you do not have the manage server permission, have somebody with it use this command instead!**")
+        state = servers[str(message.server.id)]["beta_features"]
+        if False:
```

Comment:
This will never run..

Reply:
Yeah I know. I used it for testing because your permissions were giving errors

Reply:
so should we fix this?

Reply:
if message.author.permissions.manage_server:
This line gives an error

Reply:
So I think that's the permission section on your end, so I just it to false for now